pages/home.js

```diff
@@ -2,6 +2,17 @@ import { useState } from "react";
 import Head from "next/head";
 import Button from "../components/button";
 import DashNavbar from "../components/dashNavbar";
+import { isAuthenticated, redirectHome } from "../lib/serverSideAuth";
+import MemorialCard from "../components/memorialCard";
+
+export async function getServerSideProps(ctx) {
+  const isAuth = await isAuthenticated(ctx);
+  if (!isAuth) return redirectHome(ctx);
+  return {
+    props: {},
+  };
+}
```

Comment:
we removed this in a previous PR. Try pulling from develop into your branch then push that commit

Reply:
oh you did that's weird

Reply:
..